tendrl/gluster_integration/objects/volume/atoms/generate_brick_mapping/__init__.py

```diff
@@ -34,15 +34,22 @@ def run(self):
             host = NS._int.client.read(key).value
             nodes[host] = []
 
-        bricks = NS._int.client.read(
+        hosts = NS._int.client.read(
             '/clusters/%s/Bricks/free/' % NS.tendrl_context.integration_id
         )
-        for brick in bricks.leaves:
-            brick = brick.key.split("/")[-1]
-            _brick_host = brick.split(":")[0]
-            if _brick_host in nodes:
-                if len(nodes[_brick_host]) < brick_count:
-                    nodes[_brick_host].append(brick)
+        for host in hosts.leaves:
+            host = host.key.split("/")[-1]
+            bricks = NS._int.client.read(
```

Comment:
Cant we use load_all() here at least?

Reply:
The bricks we are trying to access here are free bricks, these are not objects but are references. So I feel load all can't be used here.

Reply:
Ack